ssl/quic/quic_record_rx.c

```diff
@@ -167,8 +167,6 @@ struct ossl_qrx_st {
     SSL *msg_callback_ssl;
 };
 
-static void qrx_on_rx(QUIC_URXE *urxe, void *arg, const QUIC_CONN_ID *dcid);
-
```

Comment:
Giving deference to the fact this wasn't changed here, I noticed that ossl_qrx_new is returning 0 in a function that is typed to return a pointer to OSSL_QRX.  that should probably be fixed